### ISSUE TITLE
Nuxt 3 Example

### ISSUE THREAD

**Issue by louwers** [brillout/telefunc#46]
The [Nuxt 2 example](https://github.com/brillout/telefunc/tree/main/examples) can be replaced by a [Nuxt 3](https://v3.nuxtjs.org/) example.

@Kimbo0o ran into some issues (#41) related to Nuxt internals (which uses [vite-node](https://github.com/vitest-dev/vitest/tree/main/packages/vite-node)). Probably needs some digging and possibly an upstream fix to resolve.

**Comment by brillout** [comment]
Yes, a Nuxt 3 example would be lovely. Especially now that Nuxt 3 uses Vite.

**Comment by wobsoriano** [comment]
A working dev demo but failing in build https://github.com/wobsoriano/nuxt-telefunc-demo

https://discord.com/channels/815937377888632913/873214311957401631/1041581930249990154

**Comment by brillout** [comment]
Blocker: https://github.com/nuxt/framework/issues/8986.
